repoclosure/yum.conf

```
@@ -61,20 +54,17 @@ name=SaltStack configmanagement
 baseurl=https://repo.saltproject.io/salt/py3/redhat/8/x86_64/latest
 
 [el8-extras]
-name=extras
-mirrorlist=http://mirrorlist.centos.org/?release=8-stream&arch=x86_64&repo=extras
-#baseurl=http://vault.centos.org/centos/8/extras/$arch/os/
+name=AlmaLinux 8 - Extras
+mirrorlist=https://mirrors.almalinux.org/mirrorlist/8/extras
 
 [el8-epel]
 name=epel
 mirrorlist=http://mirrors.fedoraproject.org/mirrorlist?repo=epel-8&arch=x86_64
 failovermethod=priority
 
 [el8-powertools]
-name=powertools
-mirrorlist=http://mirrorlist.centos.org/?release=8-stream&arch=x86_64&repo=PowerTools&infra=$infra
-#baseurl=http://vault.centos.org/centos/8/PowerTools/$arch/os/
-failovermethod=priority
+name=AlmaLinux 8 - CRB
+mirrorlist=https://mirrors.almalinux.org/mirrorlist/8/crb
```

```suggestion
mirrorlist=https://mirrors.almalinux.org/mirrorlist/8/powertools
```

CRB is the EL9 name :)